woocommerce/woocommerce-ios
# [Migrate simple payments] Analytics & enable feature flag for release 18.2

Closes: #12387 
⚠️ Based on https://github.com/woocommerce/woocommerce-ios/pull/12450 for the feature flag conflicts & UI test fix


- [x] @jaclync registers new events in Tracks

## Why

Now that the feature is fully implemented in the branches this PR is based off of, analytics are implemented based on pffQ75-h8-p2#metrics with two new events on the migration sheet shown and tap events.

## How

- `simple_payments_migration_sheet_add_custom_amount`: When tapping the Add Custom Amount CTA in the migration sheet
- `simple_payments_migration_sheet_shown`: When the migration sheet is shown

## Testing instructions


- Go to the menu tab
- Tap on Payments > Collect Payment --> once the migration sheet is shown, `simple_payments_migration_sheet_shown` should be tracked in the console
- Try rotating the device --> no extra events should be tracked
- Tap `Add a Custom Amount` --> `simple_payments_migration_sheet_add_custom_amount` should be tracked in the console

---
